lib/tasks/lint.rake

```diff
@@ -1,5 +1,10 @@
 desc "Run all linters"
 task lint: :environment do
   sh "bundle exec rubocop"
+  if Rails.env.development?
+    sh "bundle exec erblint --lint-all --autocorrect"
```

Comment:
I've copied the behaviour of Collections Publisher as i don't have particularly strong feelings on whether this should autocorrect locally. Happy to go with the conensus.